frontend/openshift/templates/frontend-pre-deploy.json

```diff
@@ -0,0 +1,84 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "${NAME}-${TAG_NAME}"
+  },
+  "objects": [
+    {
+      "kind": "ConfigMap",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}-${TAG_NAME}-caddy-config",
+        "labels": {
+          "app": "${NAME}-${TAG_NAME}",
+          "app-group": "${APP_GROUP}",
+          "template": "${NAME}-pre-deploy"
+        }
+      },
+      "data": {
+        "Caddyfile": "0.0.0.0:2015\n\nroot /var/www/html\n\nlog stdout\n\nerrors stdout\n\nrewrite /${WEB_APP_CONTEXT_PATH} {\n    regexp .*\n    to {path} /${WEB_APP_CONTEXT_PATH}/\n}\n\n\n"
+      }
+    },
+    {
+      "kind": "ConfigMap",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}-${TAG_NAME}-ui-configuration",
+        "labels": {
+          "app": "${NAME}-${TAG_NAME}",
+          "app-group": "${APP_GROUP}",
+          "template": "${NAME}-pre-deploy"
+        }
+      },
+      "data": {
+        "configuration.json": "{\n  \"VUE_APP_COPS_REDIRECT_URL\": \"https://coops-${TAG_NAME}.pathfinder.gov.bc.ca/\",\n  \"VUE_APP_PAY_ROOT_API\": \"https://pay-api-${TAG_NAME}.pathfinder.gov.bc.ca/api/v1\",\n  \"VUE_APP_AUTH_ROOT_API\": \"https://auth-api-${TAG_NAME}.pathfinder.gov.bc.ca/api/v1\",\n  \"VUE_APP_STATUS_ROOT_API\": \"https://status-api-dev.pathfinder.gov.bc.ca/api/v1\", \"VUE_APP_FLAVOR\": \"mvp\"\n}\n"
```

Comment:
presumably `search-api-${TAG_NAME}.pathfinder...`

and name the variable VUE_APP_SEARCH_ROOT ?